.teamcity/src/main/kotlin/promotion/StartReleaseCycle.kt

```diff
@@ -42,16 +38,13 @@ object StartReleaseCycle : BasePromotionBuildType(vcsRootId = gradlePromotionMas
                 name = "Promote"
                 tasks = "clean promoteStartReleaseCycle"
                 useGradleWrapper = true
-                gradleParams = """-PuseBuildReceipt -PconfirmationCode=%confirmationCode% "-PgitUserName=%gitUserName%" "-PgitUserEmail=%gitUserEmail%" """
+                gradleParams = """-PcommitId=%dep.Check_Stage_ReadyforNightly_Trigger.build.vcs.number% -PconfirmationCode=%confirmationCode% "-PgitUserName=%gitUserName%" "-PgitUserEmail=%gitUserEmail%" """
```

Comment:
The id for the dependency seems to be wrong here, it needs to contain the prefix. I suppose we should use `RelativeId` here again.

Reply:
Yes, `StartReleaseCycle` is only on `master` so it can be absolute.

Reply:
Let's still use `RelativeId`, that should be easier than hardcoding the prefix.